extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -315,19 +319,43 @@
   flex-direction: row-reverse !important;
 }
 
+.i-amphtml-progress-bar-overflow.i-amphtml-story-progress-bar {
+  left: auto !important;
+  right: auto !important;
+  margin: 4px 0 0 !important;
+  padding: 0 !important;
+}
+
 .i-amphtml-story-desktop-panels .i-amphtml-story-progress-bar {
   position: relative !important;
   height: 3px !important;
-  width: calc(100vw / 3) !important;
+  width: 100% !important;
+  max-width: calc(3/5 * 75vh) !important;
```

Comment:
I copied these styles from the `amp-story-page` dimensions in the desktop panels css file so that the widths match:

https://github.com/ampproject/amphtml/blob/94227379140566c174cec3ac7f27c2838ce45e9a/extensions/amp-story/1.0/amp-story-desktop-panels.css#L122-L132

Reply:
Can't you just use `max-width: 100%`?

Reply:
Hmmm not sure I understand what you're implying. We need the `max-width` to match in both the `amp-story-page` and the `progress-bar` in the desktop panels. 

Right now, the `max-width` for the `amp-story-page` is `max-width: calc(3/5 * 75vh`, so I just changed the progress bar width in desktop panels to match that.

Reply:
Whoops, sorry, forgot that the progress bar is outside the page 😅

Reply:
In this case, I wonder: can you use `max-width: calc(100 * --story-page-vw)`?

Reply:
Ugh, actually that will cause weird jank on resize, since the custom property gets updated through the (debounced) JS resize handler.

I guess you can just do what you wrote here.

Reply:
Could we keep the current design (not copy any CSS), and measure the progress bar element instead of measuring the page?
I don't think there is any need to change the design to write this animation.